core/templates/dev/head/components/hint_editor_directive.html

```diff
@@ -54,7 +54,8 @@
 
   <div ng-if="stateHintsService.savedMemento[getIndexPlusOne() - 1] && !stateHintsService.savedMemento[getIndexPlusOne() - 1].isEmpty()">
     <audio-translations-editor component-name="<[COMPONENT_NAME_HINT]>"
-                               subtitled-html="stateHintsService.displayed[getIndexPlusOne() - 1].hintContent"
+                               content-ids-to-audio-translations="stateContentIdsToAudioTranslationsService.displayed"
```

Comment:
@seanlip, Can we remove this (also on-change) as this will be same in all other places?